staging/src/k8s.io/sample-controller/controller.go

```diff
@@ -196,50 +198,50 @@ func (c *Controller) runWorker(ctx context.Context) {
 // processNextWorkItem will read a single work item off the workqueue and
 // attempt to process it, by calling the syncHandler.
 func (c *Controller) processNextWorkItem(ctx context.Context) bool {
-	obj, shutdown := c.workqueue.Get()
+	item, shutdown := c.workqueue.Get()
 	logger := klog.FromContext(ctx)
 
 	if shutdown {
 		return false
 	}
 
 	// We wrap this block in a func so we can defer c.workqueue.Done.
-	err := func(obj interface{}) error {
+	err := func(item interface{}) error {
```

Comment:
if we eliminated this function it would match our example and existing controllers

Reply:
Yeah. I assume somebody put it here for a reason.

Reply:
Git blame tells me that this func was here from the start --- which was PR #52753 from @munnerz .

Reply:
Let's get this eliminated to match the long standing doc in https://github.com/kubernetes/community/pull/7741 that we use for kubernetes controllers.

Reply:
@deads2k: I have removed that inner func.